sw/device/silicon_creator/rom/rom_start.S

```diff
@@ -260,6 +260,20 @@ LABEL_FOR_TEST(kRomStartWatchdogEnabled)
   li   t0, SW_STRAP_2_INSEL
   sw   t0, (SW_STRAP_2_PERIPH * 4)(a0)
 
+  // Spin for a minimum of 5us worth of cycles (assuming 100MHz clock), while we
+  // wait for the RMA strap pull enables to propagate to the physical pads.
+  // Since this is a wait loop, disable the watchdog.
+  csrw mcycle, zero
+  li   t0, 500  // 500 clock cycles with 100MHz clock is 5us.
+  li   a0, TOP_EARLGREY_AON_TIMER_AON_BASE_ADDR
+  sw   zero, AON_TIMER_WDOG_CTRL_REG_OFFSET(a0)
+.L_rma_strap_pu_spin_cycles_loop:
+  csrr t1, mcycle
+  bltu t1, t0, .L_rma_strap_pu_spin_cycles_loop
+  // Reenable the watchdog timer.
+  li t1, (1 << AON_TIMER_WDOG_CTRL_ENABLE_BIT)
+  sw t1, AON_TIMER_WDOG_CTRL_REG_OFFSET(a0)
```

Comment:
Do we really want to turn the watchdog off and back on here?

Considering the watchdog is set for 1 second and this delay will be 500us to 2ms (ie: uncalibrated is 4x slower), I'm not sure its worthwhile to temporarily disable the watchdog.  @alphan, WDYT?

Reply:
The delay should be 5us to 20 ms ... 500 cycles on a 100MHz should be 5us. So we are well under the WD timer. I can remove that bit.

Reply:
fixed.

Reply:
I agree, no need to turn it off and back on. Thanks for catching this @cfrantz and thanks for the update @timothytrippel !